apps/ts-minbar-test-react-components/package.json

```diff
@@ -9,7 +9,7 @@
   },
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29281&commitId=6d333fe637209bea2831bb2de614e243871ee108&env=prod&clientType=FLUENTUI)


  <details><summary><b>TeachingBubble</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TeachingBubble.SmallHeadline.default.chromium.png|22| Changed|
|TeachingBubble.WideIllustration.default.chromium.png|23| Changed|
|TeachingBubble.WideIllustration - RTL.default.chromium.png|23| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>